subprojects/core/src/main/java/org/gradle/api/tasks/bundling/Tar.java

```diff
@@ -22,28 +22,32 @@
 import org.gradle.api.internal.file.archive.compression.GzipArchiver;
 import org.gradle.api.internal.file.archive.compression.SimpleCompressor;
 import org.gradle.api.internal.file.copy.CopyAction;
+import org.gradle.api.provider.Property;
 import org.gradle.api.tasks.Input;
-import org.gradle.internal.instrumentation.api.annotations.ToBeReplacedByLazyProperty;
+import org.gradle.internal.instrumentation.api.annotations.ReplacesEagerProperty;
 import org.gradle.work.DisableCachingByDefault;
 
 /**
  * Assembles a TAR archive.
  */
 @DisableCachingByDefault(because = "Not worth caching")
 public abstract class Tar extends AbstractArchiveTask {
-    private Compression compression = Compression.NONE;
```

Comment:
❓  This looks like it should be a convention?

Reply:
Fixed